src/components/AutoCompleteTags/AutoCompleteTags.js

```diff
@@ -0,0 +1,73 @@
+import React, { Fragment, Component } from 'react';
+import PropTypes from 'prop-types';
+import styled /* , { css } */ from 'react-emotion';
+import { remove } from 'lodash/fp';
+
+import AutoCompleteInput from '../AutoCompleteInput';
+import Tag from '../Tag';
+
+const TagsWrapper = styled('div')`
+  margin-top: ${props => props.theme.spacings.kilo};
+  /* this *hack* is to not allow the tags to be visible bellow the overlay */
+  padding: 0 1px 0 1px;
+
+  span {
+    display: inline-block;
+    margin-bottom: ${props => props.theme.spacings.byte};
+  }
+`;
+
+class AutoCompleteTags extends Component {
+  static propTypes = {
+    handleChange: PropTypes.func.isRequired,
+    availableTags: PropTypes.arrayOf(PropTypes.string).isRequired
+  };
+
+  state = { tags: [] };
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.tags.length !== this.state.tags) {
+      this.props.handleChange(this.state.tags);
+    }
+  }
+
+  handleAddTag = item =>
+    this.setState(({ tags }) => ({ tags: [...tags, item] }));
+
+  handleRemoveTag = newTag =>
+    this.setState(({ tags }) => ({
+      tags: remove(tag => tag === newTag)(tags)
+    }));
+
+  render() {
+    const { availableTags } = this.props;
+    const { tags } = this.state;
+    const autoCompleteItems = availableTags.filter(
+      item => !tags.includes(item)
```

Comment:
You need to use `includes` from lodash. Older browsers we support (including the webview of android) does not have built in support for .includes.